### PR TITLE
Feature/add models to api

### PR DESCRIPTION
**Pull request by hinkevich**

### CODE REVIEW

File: GymApp/GymAppApi/Models/CouchVisitor.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿namespace GymAppApi.Models
+{
+    public class CouchVisitor
+    {
+        public int CouchId { get; set; }
+        public Couch  Couch { get; set; }
+        public int VisitorId { get; set; }
+        public Visitor visitor { get; set; }
```
Comment (5Ka-me) [inline GymApp/GymAppApi/Models/CouchVisitor.cs]: Separate attributes from links. Add "= null!" or "?" according to relationships in database

File: GymApp/GymAppApi/Models/Order.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    public class Order
+    {
+        public int Id { get; set; }
+        public string Title { get; set; }
+        public string Description { get; set; }
+        public double Coust { get; set; }
+
+        //Navigation property
+        public int VisitorId { get; set; }
+        public Visitor Visitor { get; set; }
```
Comment (5Ka-me) [inline GymApp/GymAppApi/Models/Order.cs]: Add "= null!" or "?" according to relationships in database

File: GymApp/GymAppApi/Models/Order.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
+﻿namespace GymAppApi.Models
+{
+    public class Order
+    {
+        public int Id { get; set; }
+        public string Title { get; set; }
+        public string Description { get; set; }
```
Comment (5Ka-me) [inline GymApp/GymAppApi/Models/Order.cs]: Add "= null!" or "?" according to your vision of a model

File: GymApp/GymAppApi/Models/Visitor.cs [hunk 4]
```diff
@@ -0,0 +1,6 @@
+﻿namespace GymAppApi.Models
+{
+    public class Visitor
+    {
+    }
```
Comment (5Ka-me) [inline GymApp/GymAppApi/Models/Visitor.cs]: Add attributes or delete empty class

File: GymApp/GymAppApi/Models/Visitor.cs [hunk 5]
```diff
@@ -0,0 +1,6 @@
+﻿namespace GymAppApi.Models
+{
+    public class Visitor
```
Comment (pavelGagarin0870) [inline GymApp/GymAppApi/Models/Visitor.cs:5]: Why this model empty ?

File: GymApp/GymAppApi/Models/Couch.cs [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+{
+    public class Couch
+    {
+        public int Id { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string Describe  { get; set; }
+
+        //Navigation property
+
```
Comment (5Ka-me) [inline GymApp/GymAppApi/Models/Couch.cs]: Delete empty line

File: GymApp/GymAppApi/Models/CouchVisitor.cs [hunk 7]
```diff
@@ -0,0 +1,10 @@
+﻿namespace GymAppApi.Models
+{
+    public class CouchVisitor
```
Comment (pavelGagarin0870) [inline GymApp/GymAppApi/Models/CouchVisitor.cs]: Maybe better to rename this model to Events and maybe better to discuss data structure with your mentor

**Review by pavelGagarin0870 — APPROVED** [review]

**Review by pavelGagarin0870 — CHANGES_REQUESTED** [review]
s

**Review by stas1408123 — APPROVED** [review]

**Comment by hinkevich** [comment]
add changes

**Comment by hinkevich** [comment]
Add changes

**Comment by hinkevich** [comment]
Add models

**Pull request closed without merging**